pages/support.html

```diff
@@ -38,7 +38,7 @@ <h2>{% trans "Community Forums" %}</h2>
   <div class="halfbox_right">
     <h2>{% trans "Developer Mailing List" %}</h2>
     <p>
-      {% blocktrans with mailing_list_link="https://lists.sourceforge.net/lists/listinfo/mixxx-devel" sourceforge_archive_link="http://sourceforge.net/mailarchive/forum.php?forum=mixxx-devel" gmane_archive_link="http://news.gmane.org/gmane.comp.multimedia.mixxx.devel" %}The <a href="{{ mailing_list_link}} ">mixxx-devel</a> developer mailing list (<a href="{{ sourceforge_archive_link }}">SourceForge</a> and <a href="{{ gmane_archive_link }}">Gmane</a> archives) is the primary means of developer communication.  The mailing list is where the Mixxx team discusses upcoming changes and programming related stuff.{% endblocktrans %}
+      {% blocktrans with mailing_list_link="https://lists.sourceforge.net/lists/listinfo/mixxx-devel" sourceforge_archive_link="https://sourceforge.net/p/mixxx/mailman/mixxx-devel/" %}The <a href="{{ mailing_list_link}} ">mixxx-devel</a> developer mailing list (<a href="{{ sourceforge_archive_link }}">SourceForge</a> and <a href="{{ gmane_archive_link }}">Gmane</a> archives) is the primary means of developer communication.  The mailing list is where the Mixxx team discusses upcoming changes and programming related stuff.{% endblocktrans %}
```

Comment:
looks like the gmane_archive_link was accidentally removed here

Reply:
On purpose. That link is (or was) dead.